tanner/api.py

```diff
@@ -13,8 +13,19 @@ def __init__(self):
 
         if query == 'stats' and not params:
             result = await self.return_stats(redis_client)
-        elif query == 'stats' and 'uuid' in params:
+        elif query == 'snare-stats' and 'uuid' in params:
```

Comment:
Maybe it is suitable to think about sub-app approach? To make it easier to add api paths. http://aiohttp.readthedocs.io/en/stable/whats_new_1_1.html#sub-applications